clay.h

```diff
@@ -360,67 +608,6 @@ Clay_RenderCommandType Clay__LayoutElementTypeToRenderCommandType[] = {
     [CLAY__LAYOUT_ELEMENT_TYPE_CUSTOM] = CLAY_RENDER_COMMAND_TYPE_CUSTOM,
 };
 
-typedef enum __attribute__((__packed__)) {
-    CLAY_ATTACH_POINT_LEFT_TOP,
-    CLAY_ATTACH_POINT_LEFT_CENTER,
-    CLAY_ATTACH_POINT_LEFT_BOTTOM,
-    CLAY_ATTACH_POINT_CENTER_TOP,
-    CLAY_ATTACH_POINT_CENTER_CENTER,
-    CLAY_ATTACH_POINT_CENTER_BOTTOM,
-    CLAY_ATTACH_POINT_RIGHT_TOP,
-    CLAY_ATTACH_POINT_RIGHT_CENTER,
-    CLAY_ATTACH_POINT_RIGHT_BOTTOM,
-} Clay_FloatingAttachPointType;
-
-typedef struct
-{
-    Clay_FloatingAttachPointType element;
-    Clay_FloatingAttachPointType parent;
-} Clay_FloatingAttachPoints;
-
-typedef struct {
-    Clay_LayoutAlignmentX x;
-    Clay_LayoutAlignmentY y;
-} Clay_ChildAlignment;
-
-typedef struct {
-    float min;
-    float max;
-} Clay_SizingMinMax;
-
-typedef struct {
-    union {
-        Clay_SizingMinMax sizeMinMax;
-        float sizePercent;
-    };
-    Clay__SizingType type;
-} Clay_SizingAxis;
-
-typedef struct {
-    Clay_SizingAxis width;
-    Clay_SizingAxis height;
-} Clay_Sizing;
-
-typedef struct {
-    uint16_t x;
-    uint16_t y;
-} Clay_Padding;
-
-typedef struct {
-    float topLeft;
-    float topRight;
-    float bottomLeft;
-    float bottomRight;
-} Clay_CornerRadius;
-
-typedef struct {
-    Clay_Sizing sizing;
-    Clay_Padding padding;
-    uint16_t childGap;
-    Clay_LayoutDirection layoutDirection;
-    Clay_ChildAlignment childAlignment;
-} Clay_LayoutConfig;
-
 Clay_LayoutConfig CLAY_LAYOUT_DEFAULT = (Clay_LayoutConfig){};
```

Comment:
Website uses this too

Reply:
Fixed this here: https://github.com/nicbarker/clay/commit/5bb725d0057155200db13890a86922d393305167